public/data/souravsk.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "Sourav Kumar",
+  "type": "personal",
+  "bio": "Open Source ",
+  "avatar": "https://github.com/souravsk.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/souravsk",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/sourav_sovu_",
+      "icon": "twitter"
+    }
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/sourav-kumar-237958141",
+      "icon": "linkedin"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Learning DSA",
+      "date": "January 2022",
+      "icon": "dollar",
+      "color": "grey",
+      "description": "Learning"
+    }
+    {
+      "title": "Learning Kubernetes",
+      "date": "January 2022",
+      "icon": "dollar",
```

Comment:
Hey, the icon would be shown as a $ are you sure about that? 😅

Reply:
Sorry I didn't see that I will remove that 
After remove i have to create a new pull request